NuRadioMC/utilities/cross_sections.py

```diff
@@ -231,7 +231,7 @@ def get_nu_cross_section(energy, flavors, inttype='total', cross_section_type='c
         if type(inttype) == str:
             if inttype == 'total':
 
-                if (type(flavors) == int or type(flavors) == np.int64):
+                if isinstance(flavors, (int, np.integer)):
```

Comment:
As above

Reply:
And all other instances. Comments were strange for me ...

Reply:
@anelles sorry for not explaining better: ... (I think) if you would define the flavors as `np.int32` the previous would fail, so the `isinstance` is more general since it accepts all `np.integer` types

Reply:
now, the check is: if flavors is a standard python int or *any* `np.integer` type.

Reply:
Sorry, I copied the check over incorrectly and got another depreciation warning. My fault.